table/sst_file_writer.cc

```diff
@@ -72,19 +75,19 @@ SstFileWriter::~SstFileWriter() {
     // abandon the builder.
     rep_->builder->Abandon();
   }
-
-  delete rep_;
 }
 
 Status SstFileWriter::Open(const std::string& file_path) {
```

Comment:
We could have passed the io_priority in ::Open() instead of the constructor, I prefer passing it here, but it's up to you.